package-res/reportviewer/reportviewer-prompt.js

```diff
@@ -284,13 +247,6 @@ define([
           me.hideGlassPane();
         }
 
-        if (me.clicking) {
-          // If "Upgrading" a Change to a Submit we do not want to process the next Submit Click, if any
-          me.ignoreNextClickSubmit = (promptMode === 'USERINPUT');
```

Comment:
If I understand correctly this logic needs to avoid extra requests when we click on submit button during processing. But now I do not see a reason to save this logic because I do not find a way to click on this button during processing (glass pane exists during processing)...